main.rb

```diff
@@ -4,6 +4,7 @@
 def main
   app = App.new
   menu = Menu.new(app)
+  app.fetch_all_data
   menu.start
 end
```

Comment:
- On exit from the App the data is saved into JSON files. 💯 💯 
![Capture](https://user-images.githubusercontent.com/63972670/225299722-124d45c4-302d-43b0-8d52-118c8f9701f1.JPG)